### PR TITLE
Thoughtful post

### PR DESCRIPTION
**Pull request by elliothursh**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: thoughtful-post/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+yarn mailing
+```
+
+## See it Live
+
+https://thoughtful-post-mailing.vercel.app/
+
+## Designs
+
+https://www.figma.com/file/P0vN4dgc5MV8JKMtNWphvN/thoughtful-post-emails
```
Comment (psugihara) [inline thoughtful-post/README.md]: can you format this like this other README's?

Also maybe a note about local assets here would be useful.

File: thoughtful-post/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "yarn mailing server build && cp -r ./emails/assets ./.mailing/public",
+    "dev": "yarn mailing",
+    "dev:local-assets": "npx mailing server scaffold && ln -sf ../../emails/assets ./.mailing/public && LOCAL_ASSETS=true yarn mailing preview"
+  },
+  "dependencies": {
+    "mailing-core": "^0.8.13",
```
Comment (psugihara) [inline thoughtful-post/package.json]: update to mailing 0.8.17 for some bugfixes

File: thoughtful-post/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "thoughful-post-mailing",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "yarn mailing server build && cp -r ./emails/assets ./.mailing/public",
+    "dev": "yarn mailing",
+    "dev:local-assets": "npx mailing server scaffold && ln -sf ../../emails/assets ./.mailing/public && LOCAL_ASSETS=true yarn mailing preview"
```
Comment (psugihara) [inline thoughtful-post/package.json:9]: `npx mailing server scaffold`? I don't think that's a command.

**Review by psugihara — APPROVED** [review]
Looks great! A few comments to consider.

File: thoughtful-post/package.json (hunk 3 above)
Comment (elliothursh) [reply thoughtful-post/package.json:9]: It isn't a command, it's a hack ensure that the .mailing directory exists. You can pass in any subcommand to the mailing server command, and unless it's `build` or `start` all it does is bootstrap the mailing directory.

Comment (psugihara) [reply thoughtful-post/package.json:9]: haha o cool

**Pull request merged**